backend/cmd/start.go

```diff
@@ -172,8 +172,11 @@ func newStartCommand() *cobra.Command {
 			// Sensu APIs TLS config
 			certFile := viper.GetString(flagCertFile)
 			keyFile := viper.GetString(flagKeyFile)
-			trustedCAFile := viper.GetString(flagTrustedCAFile)
 			insecureSkipTLSVerify := viper.GetBool(flagInsecureSkipTLSVerify)
+			// TODO(gbolo): trustedCAFile will be useful in the future when TLS mutual auth is supported.
+			// Not used for backend right now, but leaving it here for now since it appears
+			// that it is being used for etcd client config
+			trustedCAFile := viper.GetString(flagTrustedCAFile)
```

Comment:
I'd favor separating the two, as explained here: https://github.com/sensu/sensu-go/issues/2548#issuecomment-462521459. That can probably be done in a separate PR though.

Reply:
I will change comment to better explain that